tests/test_migration_file.py

```diff
@@ -290,3 +290,23 @@ def test_mixed_digits_output_mode2(runner_gen, request, capfd):
     )
     output = capfd.readouterr()  # ease debug
     assert expected == tuple(output.out.splitlines())
+
+def test_allow_series_false(runner_gen, request, capfd):
+    runner = runner_gen('migration.yml', allow_serie=False)
+    with pytest.raises(
+            MigrationError,
+            match=r"[.]*Only one version can be upgraded at a time.[.]*"
+    ):
+        runner.perform()
+
+def test_allow_series_true(runner_gen, request, capfd):
+    runner = runner_gen('migration.yml')
+    try:
+        runner.perform()
+    except MigrationError as e:
```

Comment:
I don't get this test... what are you testing?

Reply:
well it tries to test that the migration runs, with multiple versions and allow_serie=True but i try to give a different message if the test fails because of some other issue.

Reply:
this test tries to make sure the migration passes if allow_serie is true. in that case there is no migration error. But since i actually need at least 2 versions which aren't noops for it to work. so there is stuff which can fail that doesn't relate to this test. so i try to give two different messages.

Reply:
@simahawk did my explanation help?

Reply:
@tschanzt my point is that you should not detect a failure. If you need another file, add it... 2 non-noop versions could for instance just print a line, nothing more.